CompassSDK/Storage/CompassStorage.swift

```diff
@@ -29,16 +34,28 @@ class PListCompassStorage: PListStorage {
         static var empty: Model {.init(numVisits: 0, userId: nil, suid: nil, firstVisit: nil, lastVisit: nil)}
     }
 
-    let filename = "CompassPersistenceV2"
-
     init() {
-        self.model = load() ?? .empty
+        self.model = loadModel() ?? .empty
+    }
+    
+    private func loadModel() -> Model? {
+        if let modelV2 = load(Store.v2.rawValue) {
+            return modelV2
+        }
+        
+        guard var modelV1 = load(Store.v1.rawValue) else {
+            return nil
+        }
+                
+        (modelV1.userId, modelV1.suid) = (modelV1.suid, modelV1.userId)
```

Comment:
[CompassSDK/Storage/CompassStorage.swift](https://github.com/Marfeel/CompassSDK-iOS/commit/29320f480330ab33116828e91699101fed121a3a#diff-82d9a92e1d933fbdde85b60da4bec93f3a7f9f8f56c8bf2b8d01fda67630aa37L12)

userId and suid were stored with swapped keys. Adding backward compatibility